### ISSUE TITLE
Logo in result download missing <figure> tag and alt text

### ISSUE THREAD

**Issue by lauraykerr** [CDCgov/prime-simplereport#4006]
## Area or Component
Result download 

## Quick Summary
The SR logo at the top of the result download page isn't tagged and has no alt text.

## Accessibility Violation
Logo images in PDFs should be tagged `<figure>` and given a descriptive alt text so they can be read by screen readers.

## Action Requested
If possible when generating a result PDF:
-  Include `<figure>` tag for SR logo
- Include the alt text `"SimpleReport logo"`
